connect/connect-c8/element-template-c8/src/main/java/io/miragon/miranum/connect/adapter/in/c8/elementtemplates/schema/Binding.java

```diff
@@ -0,0 +1,291 @@
+     */
+    @JsonProperty("type")
+    @JsonPropertyDescription("The type of a property binding")
+    private Binding.Type type;
+
+    /**
+     * binding name
+     * <p>
+     * The name of a property binding
+     */
+    @JsonProperty("name")
+    @JsonPropertyDescription("The name of a property binding")
+    private String name;
+
+    /**
+     * binding source
+     * <p>
+     * The source value of a property binding (zeebe:output)
+     */
+    @JsonProperty("source")
+    @JsonPropertyDescription("The source value of a property binding (zeebe:output)")
+    private String source;
+
+    /**
+     * binding key
+     * <p>
+     * The key value of a property binding (zeebe:taskHeader)
+     */
+    @JsonProperty("key")
+    @JsonPropertyDescription("The key value of a property binding (zeebe:taskHeader)")
+    private String key;
+
+    /**
+     * binding type
+     * <p>
+     * The type of a property binding
+     * (Required)
+     */
+    @JsonProperty("type")
+    public Binding.Type getType()
+    {
+        return type;
+    }
+
+    /**
+     * binding type
+     * <p>
+     * The type of a property binding
+     * (Required)
+     */
+    @JsonProperty("type")
+    public void setType(Binding.Type type)
+    {
+        this.type = type;
+    }
+
+    public Binding withType(Binding.Type type)
+    {
+        this.type = type;
+        return this;
+    }
+
+    /**
+     * binding name
+     * <p>
+     * The name of a property binding
+     */
+    @JsonProperty("name")
+    public String getName()
+    {
+        return name;
+    }
+
+    /**
+     * binding name
+     * <p>
+     * The name of a property binding
+     */
+    @JsonProperty("name")
+    public void setName(String name)
+    {
+        this.name = name;
+    }
+
+    public Binding withName(String name)
+    {
+        this.name = name;
+        return this;
+    }
+
+    /**
+     * binding source
+     * <p>
+     * The source value of a property binding (zeebe:output)
+     */
+    @JsonProperty("source")
+    public String getSource()
+    {
+        return source;
+    }
+
+    /**
+     * binding source
+     * <p>
+     * The source value of a property binding (zeebe:output)
+     */
+    @JsonProperty("source")
+    public void setSource(String source)
+    {
+        this.source = source;
+    }
+
+    public Binding withSource(String source)
+    {
+        this.source = source;
+        return this;
+    }
+
+    /**
+     * binding key
+     * <p>
+     * The key value of a property binding (zeebe:taskHeader)
+     */
+    @JsonProperty("key")
+    public String getKey()
+    {
+        return key;
+    }
+
+    /**
+     * binding key
+     * <p>
+     * The key value of a property binding (zeebe:taskHeader)
+     */
+    @JsonProperty("key")
+    public void setKey(String key)
+    {
+        this.key = key;
+    }
+
+    public Binding withKey(String key)
+    {
+        this.key = key;
+        return this;
+    }
+
+    @Override
```

Comment:
Do we need a toString()?

Reply:
Propably not I just created them because the generated POJOs for C7 also have it.